rke/templates/templates.go

```diff
@@ -160,7 +160,7 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.8.0-rancher0 <1.13.0-rancher0":        canalv18,
 		},
 		kdm.Flannel: {
-			">=1.25.0-rancher-1-1":                     flannelv0192,
+			">=1.25.0-rancher1-1":                      flannelv0192,
 			">=1.24.10-rancher1-1 <1.25.0-rancher-1-1": flannelv0140Rancher2,
```

Comment:
Should probably change the top range of the previous range too (may not cause any issues, but should be fixed for consistency.

```suggestion
			">=1.24.10-rancher1-1 <1.25.0-rancher1-1": flannelv0140Rancher2,
```

Reply:
thanks for pointing it out. fixed it.